index.html

```diff
@@ -2,38 +2,70 @@
 <head>
 <title>Chef Brian Bevand</title>
 <style type="text/css">
-table, h1
-{
-	border: 0px solid black;
-	border-collapse:collapse;
-	width: 100%;
-}
-td
-{
-	border: 0px;
-	border-collapse:collapse;
-	width: 25%;
+body{ 
+    background-image: url("veggies.jpg");
+    background-position: center;
+    background-size: cover;
+	font-family: sans-serif;
+	color: white;
 }
 img {
 	max-width: 100%;
 	height: auto;
+	opacity: 100%;
+}
+.title {
+	font-family:impact;
+	color: white;
+	font-size: 60px;
+}
+.extra-text {
+  display: inline-block;
+}
+footer {
+	font-size: 10px;
+	text-align: center;
+}
+.food {
+	width: 85%;
```

Comment:
Is this indentation done with tabs or spaces?

While there may or may not be common use of one vs. the other in certain communities, when contributing to an existing project, it's usually best to follow whatever convention is already there unless there is a strong reason not to.

Reply:
I created the "food" thing for my table's background

Reply:
Please re-read the previous comment.